README.md

```diff
@@ -97,19 +112,17 @@ interface FolderBlockProps {
 
 File Blocks can implement editable content by calling `onUpdateContent` to update the current content. The Blocks application has a Save button which prompts the user to accept the change and create a new commit (or branch) in the repo. When content has been updated but not yet committed, the `originalContent` prop contains the original file content, so the block can show a diff. When the `isEditable` flag is false, the user does not have permission to edit the file, so the block should disable editing.
 
+Blocks can be nested inside other blocks using `BlockComponent`; you can
+retrieve a list of available blocks using `onRequestBlocksRepos`. See the
+[edit-and-preview](https://github.com/githubnext/blocks-examples/blob/main/blocks/file-blocks/edit-and-preview/index.tsx)
+block for an example of their use.
+
 A few caveats and callouts:
 
 - You can import CSS files or split your block into multiple TypeScript files, and everything will be bundled with your block.
 - You can use any third-party dependencies from NPM; just add them with `yarn add` and import them as usual, and they'll be bundled with your block.
 - The [GitHub Primer React components](https://primer.style/react/) are already included as a dependency
 - Your Block entry file **must have the Block component as its default export** so the GitHub Blocks application can find it.
-- To make authenticated requests to the GitHub API, create a [personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) (with `repo` scope) and pass it when you start the dev server:
```

Comment:
I just realized that this isn't needed anymore - love it!

Reply:
yeah it is nice! much simpler